20230728/s16-1690552431770/README.md

```diff
@@ -0,0 +1,72 @@
+# Scenario 16 - ALS Baseline with Sims-only, Disabled JSON.stringify [ALS v14.2.3](https://github.com/mojaloop/account-lookup-service/releases/tag/v14.2.3) + Scale 4 + 6x k6 VUs + Enhanced Caching for ValidateParticipants + OracleRequest | 1 | N | [account-lookup-service/pull/461](https://github.com/mojaloop/account-lookup-service/pull/461), [v14.3.0-snapshot.0](https://github.com/mojaloop/account-lookup-service/releases/tag/v14.3.0-snapshot.0)
```

Comment:
What is different in this scenario from scenario 15 apart from different ALS version? It has Oracle Request. Does this mean there is Oracle lookup involved?

Reply:
Correct. 

The Oracle Request is being cached.